surveys/2024-annual-survey/questions.md

```diff
@@ -0,0 +1,890 @@
+> to better understand the shape of community involvement and for cohort analysis.
+
+### What has your experience been like in the following Rust community spaces?
+
+Type: matrix
+
+Community Spaces:
+
+- *Official* Rust community forums or chats (users.rust-lang.org, internals.rust-lang.org, the official Rust Discord, or the Rust Zulip)
+- *Unofficial* Rust community forums or chats (e.g., reddit.com/r/rust, Hacker News, the Rust *Community* Discord, etc.)
+- Community focused on a specific area of Rust software development (e.g. game development, audio, etc.)
+- Rust conferences
+- Rust meetups or local community events
+- Discussions (issues, pull requests, etc.) on a repository *inside* the rust-lang GitHub organization
+- Discussions (issues, pull requests, etc.) on a Rust repository *outside* of the rust-lang GitHub organization
+
+Choices:
+
+- I feel welcome
+- I do not feel particularly welcome or unwelcome
+- I feel unwelcome
+- I've never participated in this activity
+
+> **justification**
+>
+> We'd like to know where people are feeling welcome and the degree to which they are feeling welcome.
+
+
+### If you indicated that you did not feel welcome in the Rust community, are there any details about your experience that you would like to share with us?
+
+Type: free form (optional)
+
+> **justification**
+> 
+> More detail on the type of situations where people have felt unwelcome can let us better 
+> address these issues in the future.
+
+## Rust in Education
+
+### Are you currently taking a course that uses or teaches Rust OR have you taken a course of this type in the last year?
+
+Type: select one (optional)
+
+- Yes, through a university, school, or other educational institution
+- Yes, through my employer, contractor, or consultancy
+- No
+
+> **justification**
+>
+> This question is primarily used to funnel respondents into the more specific questions about the kinds of educational activities they've been a part of.
+
+### If you consumed learning material about Rust, which kind of material did you consume?
+
+Type: select all that apply (optional)
+
+- Books ("The Rust Programming Language", "Rust for Rustaceans", etc.)
+- Online exercises (Rustlings, Rust by Example, etc.)
+- Videos
+- Online courses, webinars
+- Other (please specify)
+
+> **justification**
+> 
+> Getting data here seems helpful for guiding users / recommending public content.
+
+## Rust at work
+
+### Are you employed full- or part-time (including paid internships)?
+
+Type: select one
+
+- Yes
+- No [`NEXT`](#your-opinions-about-rust)
+
+### Do you write or design software in your work?
+
+Type: select one (optional)
+
+- Yes, primarily as an individual contributor (i.e., non-manager)
+- I primarily manage others who do
+- No [`NEXT`](#to-what-extent-is-rust-currently-being-used-by-your-company)
+
+### Are you personally using Rust at work?
+
+Type: select one
+
+- Yes, for the majority of my coding
+- Yes, but I only use it occasionally
+- No
```

Comment:
```suggestion
- Yes, for the majority of my coding
- Yes, weekly (in average)
- Yes, but I only use it occasionally
- No
```

closes #252

Reply:
Reworded to "Yes, a few times per week on average", unless you object :)